### PR TITLE
Routes & folder cleanup

### PR DESCRIPTION
**Pull request by michael-martin**
- Swap back to JSX route definitions
- Simplify creation of pages (A simple folder under route for each page, which should amalgamate the relevant pieces from `components` for the current page)
- Moves `/images/` into `/static/`
- Delete some starter kit boilerplate

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -1,36 +1,11 @@
...
-    inside the route `getComponent` function, since it is only invoked
-    when the route exists and matches.
-*/
-
-export default createRoutes
+export default (
+  <Route path="/" component={CoreLayout}>
+    <IndexRoute component={Home} />
+  </Route>
+);
```
Comment (vitkon) [inline src/routes/index.js:11]: 👍

**Comment by vitkon** [comment]
can we clean up async reducers in the same PR as it's all part of the same code splitting boilerplate?

**Comment by michael-martin** [comment]
I'd rather merge this and get an end-to-end feature in place, then come back later to look at the async reducers. We have the issue raised to tackle it

(Just pretty conscious that getting in a first example feature is holding up some of the other PRs too. We'll get to the async piece by Thursday?)

**Comment by vitkon** [comment]
OK, lgtm then

**Pull request merged**